module/zcommon/zfs_prop.c

```diff
@@ -865,17 +865,35 @@ zfs_prop_align_right(zfs_prop_t prop)
 #endif
 
 #if defined(_KERNEL)
+
+#if defined(HAVE_KERNEL_FPU_INTERNAL)
+#include <linux/simd.h>
+
+struct fpu **zfs_kfpu_fpregs;
+EXPORT_SYMBOL(zfs_kfpu_fpregs);
```

Comment:
same as above, the fpu struct is more than just the saved fpregs state..

Reply:
Fixed